### DISCUSSION TITLE
Why does our community SPL token show with collectibles and not on the main tab?

### DISCUSSION THREAD

**Discussion by hey-its-slowly** [phantom/docs#137] [Help & Support]
### Summary

Our zero-decimal community token has always shown up on the "collectibles" tab of Phantom wallet, and I cannot figure out why.

This is the token: https://solscan.io/token/AD2AWgBPdfbvko4a7i6ZjKq1JxDGRUTPWpd7bUJ2hSaK#metadata

I asked a few times in the old Phantom Discord, but haven't been able to get an answer.

I've tried:

- double-checking the metadata format as described in the [phantom docs](https://docs.phantom.app/best-practices/tokens#categorizing-tokens)
- checked that it was set up correctly using the official method described [here](https://spl.solana.com/token)
- compared it to [a similar zero-decimal token (FOXY)] (https://solscan.io/token/FoXyMu5xwXre7zEoSvzViRk3nGawHUp9kUh97y2NDhcq#metadata) and the metadata appears to be in the correct format

Screenshot:

![CleanShot 2023-12-13 at 13 41 18@2x](https://github.com/phantom/docs/assets/134097107/ae24526a-9f90-4645-841f-44bb843c87da)

Can anyone tell me the difference?

### Steps to Reproduce

Create an spl-token and mint some to your wallet.

Set metadata with the following format: 

```json
{
  "name": "$ROUNDIE",
  "symbol": "ROUNDIE",
  "description": "$ROUNDIE is the next-generation currency for Roundie Town, the community inhabited by Roundies on the Solana blockchain.",
  "image": "https://shdw-drive.genesysgo.net/Hex2L88BsxgLKkjNWLhziNzQDXNQkhJrEzZudWsPyC3m/ROUND.png",
  "external_url": "https://theroundies.co"
}
```

The token will show in the collectibles page.

### Phantom Version

All versions

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @hey-its-slowly,

This is being classified as a collection do to the metadata indicating it is a FungibleAsset. In order for it to show with other fungibles its metadata will need to be declared as a Fungible.

https://developers.metaplex.com/token-metadata/token-standard

**Reply by hey-its-slowly** [reply]
Thanks so much for the reply! Would you mind sharing, as to why [FOXY](https://solscan.io/token/FoXyMu5xwXre7zEoSvzViRk3nGawHUp9kUh97y2NDhcq#metadata) (linked above) shows in the main tab, since it's also declared as a fungible asset?

**Reply by adamdelphantom** [reply]
It's possible the default behavior was overridden at some point. But we've been avoiding that more recently as much as possible.

**Reply by hey-its-slowly** [reply]
I see, ok, thank you. I will research if it's possible to update the token asset type, and post my findings here, then I'll mark this as resolved.

**Reply by cosimo-rip** [reply]
We’ve found this to be impossible to update from our side. We have the same issue with $SKULL (SKu11EypaFU3gvr8VSAbi13zEC2CPvqbz9s83N3tWHM)

We’ve also noticed a possibly related bug where the quantities of these tokens are incorrect in the UI: https://x.com/cosimo_rip/status/1733505666575421777

Is there any chance this can be addressed from phantom’s side @adamdelphantom?
